go.mod

```diff
@@ -23,8 +23,8 @@ require (
 	github.com/lithammer/dedent v1.1.0
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/mitchellh/hashstructure v1.0.0
-	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.20.1
+	github.com/onsi/ginkgo/v2 v2.5.0
```

Comment:
we use v2.8.1 in gloo/solo-projects, should we upgrade to that?

Reply:
I don't see why not...the dep that led to requiring ginko v2 was solo-kit which [uses v2.5.0](https://github.com/solo-io/solo-kit/blob/main/go.mod#L23) so that's what I've been using elsewhere as well